docs/source/general/registry.rst

```diff
@@ -0,0 +1,87 @@
+########
+Registry
+########
+
+.. _registry:
+
+********************
+Available Registries
+********************
+
+Registries are Flash internal key-value database to store a mapping between a name and a function.
```

Comment:
Is it only meant to store backbones? If soo, let's be explicit about it.

Reply:
Done !